nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/ClusterCommandHelper.java

```diff
@@ -436,4 +430,8 @@ private static String listToString(List<String> slist) {
         public final List<String> succeededServerNames = new ArrayList<>();
         public final List<String> failedServerNames = new ArrayList<>();
     }
+
+    public void setAdminTimeout(long adminTimeout) {
```

Comment:
Some javadoc stating that this is in milliseconds wouldn't go amiss since the timeout on the actual commands is in seconds.